cmd/eksctl-anywhere/cmd/applypackages.go

```diff
@@ -42,7 +51,12 @@ var applyPackagesCommand = &cobra.Command{
 }
 
 func applyPackages(ctx context.Context) error {
-	kubeConfig := kubeconfig.FromEnvironment()
+	kubeConfig := lpo.kubeConfig
```

Comment:
Sections like this one are very repetitive. I'll be introducing some DRYer code in an upcoming PR. Just trying to keep things compartmentalized a bit.